cypress/integration/remote-storage/configurations/crud.spec.js

```diff
@@ -0,0 +1,127 @@
+import { bigtestGlobals } from '@bigtest/globals';
+import { Link } from '@bigtest/interactor';
```

Comment:
Should we re-export `Link` interactor from stripes testing?

Reply:
I don't see it there...

Reply:
@JohnC-80 ☝🏻 

Is making `Link` a stripes interactor a good idea so that nobody needs to import directly from `@bigtest/interactor`?